### PR TITLE
notes functionality update

### PR DESCRIPTION
**Pull request by vidhi-thakur**
- using local storage for saving notes
- updated add notes functionality
- added delete and archive functionality
- refractored code

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -30,6 +30,18 @@ function App() {
...
+              <Archive />
+            </Sidebar>
+          </RequiresAuth>} />
+        <Route path='/trash' element={
+          <RequiresAuth>
+            <Sidebar isSidearVisible={isSidearVisible}>
+              <Trash />
+            </Sidebar>
+          </RequiresAuth>} />
         <Route path="*" element={<h1 style={{ margin: "5rem auto", textAlign: "center" }}>Page not found</h1>} />
```
Comment (HeyNitin) [inline src/App.js:45]: try to refrain from using inline styling, even for this you could give className="error-page" and move the styling in it

File: src/pages/archive/Archive.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { NotesCard, NotesWithCategory } from 'components/componentExport'
+import React from 'react'
+import "./Archive.css"
```
Comment (HeyNitin) [inline src/pages/archive/Archive.js:3]: why are you using relative export here?  as you've used absolute imports everywhere else

File: src/pages/dashboard/Dashboard.js [hunk 3]
```diff
@@ -1,44 +1,25 @@
-import { AddNotes, DashboardCard } from 'components/dashboard/dashboardExports'
-import React from 'react'
+import React, { useState } from 'react'
 import './Dashboard.css'
```
Comment (HeyNitin) [inline src/pages/dashboard/Dashboard.js:2]: again relative import. Unless there's a particular reason, try to keep code consistent with one of the approaches (absolute imports are usually preferred)

File: src/pages/trash/Trash.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import { NotesCard, NotesWithCategory } from 'components/componentExport'
+import { getDeletedNotes } from 'helpers/helperExport'
+import React from 'react'
+import "./Trash.css"
```
Comment (HeyNitin) [inline src/pages/trash/Trash.js:4]: same

**Review by HeyNitin — APPROVED** [review]
Left some minor remarks, just for code consistency. Looks good to merge.

**Review by Amandeep-sinha — APPROVED** [review]

**Comment by vidhi-thakur** [comment]
> Left some minor remarks, just for code consistency. Looks good to merge.

thanks for pointing out. Have used relative import for all the css files and would update all the css file imports in next iteration

**Pull request merged**